hw1/task02/task02.py

```diff
@@ -0,0 +1,27 @@
+"""
+Given a cell with "it's a fib sequence" from slideshow.
+
+please write function "check_fib", which accepts a Sequence of integers, and
+returns if the given sequence is a Fibonacci sequence
+We guarantee, that the given sequence contain >= 0 integers inside.
+"""
+
+
+def check_fib(data: list) -> bool:
```

Comment:
Let try to write it in a different way. Too many ifs make it hard to read.

Reply:
Will rewrite it with len and in check